tests/libxc/dft-anion/input.dat

```diff
@@ -0,0 +1,121 @@
+memory 1 Gb
+                                           
+molecule h2o {                             
+   -1 2                                    
+   O                                       
+   H 1 1.0                                 
+   H 1 1.0 2 104.5                         
+   units angstrom                          
+   no_reorient                             
+}                                          
+                                           
+set {                                      
+   reference uhf                           
+   basis 6-31G                             
+   dft_radial_points 100                   
+   dft_spherical_points 302                
+   guess sad                               
+   d_convergence 11                        
+}                                          
+
+data = {                                   #TEST
+      'BMK':  -76.1817717171999931,        #TEST
+ 'MGGA_MS1':  -76.2292773247000071,        #TEST
```

Comment:
origin of all these nice references? at least program, preferably in/out files and options, too.